src/materialColorScheme.scss

```diff
@@ -136,3 +136,6 @@ $service-color: #ff6c44;
 
 // Color used to indicate success in general (of an operation, verified badge, etc).
 $accent-3-dark: #00bfa5 !important;
+
+// This color is commonly used by headers, titles, and names
+$header-color: #151c3d;
```

Comment:
is the in the color palette in zeplin?

Reply:
No it's not, but with how often it's used, I wonder if it should be